app/controllers/v1/things_controller.rb

```diff
@@ -1,9 +1,5 @@
-class V1::ThingsController <ApplicationController
+class V1::ThingsController < ApplicationController
   def index
```

Comment:
The app does not show any greeting. The main page is totally blank. Please clone the repo and run it again. Kindly make sure that the page shows a random greeting.